experimental/sql/src/alg_to_ssa.py

```diff
@@ -57,9 +57,10 @@ def find_type_in_parent_operator(self, name: str,
     parent_op = op.parent_op()
     while (parent_op and not isinstance(parent_op, ModuleOp)):
       if isinstance(parent_op, RelSSA.Operator):
-        type_ = parent_op.operands[0].typ.lookup_type_in_schema(name)
-        if type_:
-          return type_
+        for o in parent_op.operands:
```

Comment:
Update function documentation accordingly.

Reply:
I am realizing that this function makes the assumption that the names it should look up (1) come from the inputs and (2) are distinct across the inputs. Is this always the case? In the natural join, it's not.

In other words, I think the function doesn't actually look up the names in the parent because which names exist in the parent may be operator-specific; it rather looks them up in the inputs of the parent.

I don't remember exactly what we need this function for so I don't have a good suggestion at the moment but I have the feeling that the current code isn't particularly elegant...

Reply:
This function is mainly used to rewrite columns since we need to infer types here. The distinctness is something I am currently assuming. I have added several checks that this is the case, but I don't see how to nicely check this on this level atm. I feel our handling of columns will change at some point. Maybe removing the table form the column node was a bad idea after all...

Anywho, renamed and updated the docu.